Dockerfile

```diff
@@ -46,10 +30,6 @@ RUN curl -sLo cue.tgz https://github.com/cue-lang/cue/releases/download/v0.4.2/c
   tar -xf cue.tgz -C /usr/local/bin cue && cue version
 
 # kapp-controller
-COPY . .
-# helpful ldflags reference: https://www.digitalocean.com/community/tutorials/using-ldflags-to-set-version-information-for-go-applications
-RUN CGO_ENABLED=0 GOOS=linux go build -mod=vendor -ldflags="-X 'main.Version=$KCTRL_VER' -buildid=" -trimpath -o controller ./cmd/main.go
```

Comment:
still want to keep this last to avoid redownloading binaries when only source changes.